src/Alternative/index.js

```diff
@@ -0,0 +1,8 @@
+
+//. ### Alternative
+
+export { and } from './and'
+export { empty } from './empty'
+export { or } from './or'
+export { toBoolean } from './toBoolean'
+export { xor } from './xor'
```

Comment:
Do you expect people to continue to write `const S = require('sanctuary');` or to cherry-pick the functions of interest?

What's the reason for favouring `src/${category}/${function}.js` over `src/${function}.js`? Having functions listed in some sort of order in the readme and on the website is useful, but the categories do feel arbitrary at times. There are many functions which could justifiably be included in multiple categories. I'm reluctant to make the categories part of the API, as they will be if we support `require('sanctuary/Alternative/or')`.